recipes/pycoare/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pycoare" %}
+{% set version = "2023.10b1" %}
```

Comment:
There isn't a stable release and I believe this one falls under the exception in CFEP-05.